ScanExtensions.php

```diff
@@ -147,7 +147,7 @@ private function processFile($file) {
 	 * @return mixed|null
 	 */
 	private function queryCodeSearch( $class ) {
-		$url = 'https://codesearch.wmflabs.org/extensions/api/v1/search?';
+		$url = 'https://codesearch.wmflabs.org/things/api/v1/search?';
```

Comment:
Even though this will come up in the codesearch results, ive hard coded the /extensions into the url in $baseUrl, so that will have to change too i think!

Reply:
fixed